releasenotes/notes/params_without_schedule-20555d98875a626b.yaml

```diff
@@ -0,0 +1,11 @@
+---
+fixes:
```

Comment:
This should be upgrade note.

Reply:
Done here: [7d8a6ed](https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1202/commits/7d8a6edc8ab87ee912f47e9406eac9292658680c)